### PR TITLE
Department specific category listing API

### PR DESCRIPTION
**Pull request by Addy-tea-ya**
…es of department and appropriate routes

### CODE REVIEW

**Comment by Addy-tea-ya-official** [comment]
Added test case for show categories please check

File: spec/acceptance/v1/departments_spec.rb [hunk 1]
```diff
@@ -48,4 +51,56 @@
       end
     end
   end
+
+  get 'departments/:department_id/categories' do
```
Comment (Selectus2) [inline spec/acceptance/v1/departments_spec.rb:55]: update this as per the route changes done in routes.rb

**Review by Selectus2 — CHANGES_REQUESTED** [review]

File: spec/acceptance/v1/departments_spec.rb (hunk 1 above)
Comment (Addy-tea-ya-official) [reply spec/acceptance/v1/departments_spec.rb:55]: It is already consistent with routes.rb, all the test cases pass

For your reference I am providing the route I got using rails routes:
department_categories GET    /departments/:department_id/categories(.:format)                                                  api/v1/departments#show_categories

Comment (Addy-tea-ya-official) [reply spec/acceptance/v1/departments_spec.rb:55]: shall I add the / before departments there?

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,9 @@
       resources :tickets, only: [:create]
       resources :sessions, only: :create
       resources :departments, only: :create
+      resources :departments, only: :show do
+        get '/categories', to: 'departments#show_categories'
+      end
       get '/organizations/:id/departments', to: 'organizations#show_departments'
```
Comment (Nandini2806) [inline config/routes.rb:11]: Change this route also as member route of organization

**Review by Selectus2 — APPROVED** [review]
LGTM

**Pull request merged**